rules/react/walmart.js

```diff
@@ -16,15 +16,15 @@ module.exports = {
     // Enforce boolean attributes notation in JSX
     "react/jsx-boolean-value": 2,
     // Validate closing bracket location in JSX
-    "react/jsx-closing-bracket-location": [2, 'tag-aligned'],
+    "react/jsx-closing-bracket-location": [2, "tag-aligned"],
```

Comment:
Is there a reason we use this rule? IMO the closing bracket looks way better on the last props line.